.pre-commit-config.yaml

```diff
@@ -29,11 +29,11 @@ repos:
       - id: end-of-file-fixer
       - id: trailing-whitespace
         args: [--markdown-linebreak-ext=md]
-  - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.812
-    hooks:
-      - id: mypy
-        language_version: python3
+  # - repo: https://github.com/pre-commit/mirrors-mypy
+  #   rev: v0.812
+  #   hooks:
+  #     - id: mypy
+  #       language_version: python3
```

Comment:
Should this be included?

note you can run `SKIP=mypy git commit` if you have to..

Reply:
Agree with @vdboor 

And besides the `follow_imports = silent` should address most annoyances. (unless you're working on `types.py`).

Reply:
Yes. I'll re-enable mypy in next pull request, which will include fixes for most (if not all) mypy errors in repo. Otherwise there's no reason to do skips during commit.